client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -108,6 +108,27 @@
                 ]});
                 break;
 
+            case 'deskOutput':
+                var desk_id = card._id.substring(0, card._id.indexOf(':'));
```

Comment:
This could use a helper utility function, too, as mention in an earlier comment (extracting desk ID from the content item's ID).